src/typings-new-format/spec-1.0/methods.ts

```diff
@@ -89,14 +113,10 @@ export interface MethodBase10 extends Operation10 {
      * A method's body is defined in the body property as a hashmap, in which
      * the key MUST be a valid media type.
      */
-    body?: { [key: string]: datamodel.TypeReference10 }
+    body?:  datamodel.TypeReference10[]
```

Comment:
Here we'll have a problem with replacing `key` with name. Type declaration contains a `name` property on it's own and it is name of the type. Here use of `name` would be ambiguous because it either means type name or body media type.